generator-input/tweaks/global.json

```diff
@@ -1,6 +1,6 @@
 {
   "sdk": {
-    "version": "8.0.413",
+    "version": "8.0.415",
```

Comment:
@jskeet This feeds into the librarian image, correct? So we need to wait for that to be updated?

Reply:
We'll need to merge this, build a new image, then update the state file to use the new image. I'll do it on Monday and write it all down. (We have a process written down but it may be out of date.)